api/globalSearch/index.ts

```diff
@@ -72,3 +85,9 @@ export function generateServices(jurisdictions: Jurisdiction[]): GlobalSearchSer
   // Return generated global search services
   return globalSearchServices;
 }
+
+function toTitleCase(serviceName: string): string {
+  return serviceName.replace(/([a-zA-Z])([a-zA-Z]*)/g, (match, firstLetter, rest) => {
+    return firstLetter.toUpperCase() + rest.toLowerCase();
```

Comment:
That won't work for strings that have non alpha chars, such as "andy_wilkins", but I guess it doesn't matter